CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Unversioned
-
+- Minor: Added quotation marks around the automod phrases for clairty 
 ## 2.3.5
```

Comment:
```suggestion
- Minor: Added quotation marks in the permitted/blocked Automod messages for clarity. (#3654)

## 2.3.5
```
Also no worries on the ✅ - it's basically cosmetic and maintainers can press it if they care enough.

Reply:
Thanks for clearing that up!